### PR TITLE
Add Arch Linux AUR package to README.md

### PR DESCRIPTION
**Pull request by victorhaggqvist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ Install KeePassX from the Red Hat (or CentOS) repository:
 sudo yum install keepassx
 ```
 
+### Arch Linux
+
+Install KeePassX from source via AUR, [keepassx-git](https://aur.archlinux.org/packages/keepassx-git).
+
+```bash
+yaourt -S keepassx-git
```
Comment (inglor) [inline README.md]: Now everyone is using yaourt. Please don't suggest yaourt as an AUR helper. Archlinux users that use AUR already know how to install it so providing a bash solution which uses yaourt is not necessary.

Comment (victorhaggqvist) [inline README.md]: Fair enough, removed the `yaourt` part.

**Comment by ivoshm** [comment]
Boosting AUR package **keepass 2** over **keepassx-git* was IMHO better choice ... it's based on released source code than on git repo.

**Comment by victorhaggqvist** [comment]
Maybe, I went with `keepassx-git` since it has more votes.

**Comment by debfx** [comment]
I'll replace this section with a distro generic one instead since I assume people know how to search for and install packages in their distro.

**Comment by llybin** [comment]
community/keepassx 0.4.4-1 - https://www.keepassx.org/downloads/0-4 - old 0.*
community/keepassx2 2.0-3 - https://www.keepassx.org/releases/2.0/keepassx-2.0.tar.gz - always stable 2.*
aur/keepassx-git 2.0.beta2.r83.g0c36c40-1 - source this rep. master always 0.* -> 2.* ->

**Comment by llybin** [comment]
@victorhaggqvist Can you improve README?

### Arch Linux
Install KeePassX from the Arch Linux repositories:

```bash
sudo pacman -S keepassx2 # (stable version)
yaourt -S keepassx-git # (source code, the latest version)
```

**Comment by victorhaggqvist** [comment]
Considering what @debfx has planned I won't do any more here, closing this.

**Pull request closed without merging**